src/components/MentorsList/MentorsList.js

```diff
@@ -1,38 +1,32 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import classNames from 'classnames';
 import InfiniteScroll from 'react-infinite-scroller';
 import Card from '../Card/Card';
 
 import './MentorList.css';
 import { Loader } from '../Loader';
 
-const itemsInPage = 20;
-
 const MentorsList = props => {
-  const [page, setPage] = useState(1);
-  const [ready, setReady] = useState(false);
+  const { mentors, className, mentorCount } = props;
 
-  useEffect(() => {
-    setPage(1);
-    setReady(props.ready);
-  }, [props.mentors, props.ready]);
+  const [isReady, setIsReady] = useState(false);
 
-  const loadMore = () => {
-    setPage(page + 1);
+  const loadMore = async page => {
+    setIsReady(false);
+    await props.getMentorsPage(page);
+    setIsReady(true);
```

Comment:
I think that this will not work as expected, but I think we will not need the isReady flag anyway, we shouldn't have an empty list of mentors